server/src/routes/api/v1/reviewRouter.js

```diff
@@ -0,0 +1,23 @@
+import express from "express"
+import { Review } from "../../../models/index.js"
+import { ValidationError } from "objection";
+import cleanUserInput from "../../../services/cleanUserInput.js";
+
+const reviewRouter = new express.Router({ mergeParams: true })
```

Comment:
the name of this router and the file needs to reflect that it is specifically an `activityReviewsRouter`, not just regular reviews - reviews for activities